src/Symfony/Component/Asset/UrlPackage.php

```diff
@@ -62,7 +61,7 @@ public function __construct($baseUrls = array(), VersionStrategyInterface $versi
         $sslUrls = $this->getSslUrls($baseUrls);
 
         if ($sslUrls && $baseUrls !== $sslUrls) {
-            $this->sslPackage = new UrlPackage($sslUrls, $versionStrategy);
+            $this->sslPackage = new self($sslUrls, $versionStrategy);
```

Comment:
Fabbot.io forced to me to change me to `new self` to make the build pass. Looks like that's one of our coding conventions :)